addons/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/handler/SqueezeBoxServerHandler.java

```diff
@@ -839,14 +869,134 @@ public void updateListener(SqueezeBoxPlayerEventListener listener) {
                 } else if (function.equals("volume")) {
                     final int volume = (int) Double.parseDouble(value);
                     updatePlayer(new PlayerUpdateEvent() {
+
                         @Override
                         public void updateListener(SqueezeBoxPlayerEventListener listener) {
                             listener.absoluteVolumeChangeEvent(mac, volume);
                         }
+
                     });
                 }
             }
         }
+
+        private void handleFavorites(String message) {
+            logger.trace("Handle favorites message: {}", message);
+
+            String[] messageParts = message.split("\\s");
+            if (messageParts.length == 2 && "changed".equals(messageParts[1])) {
+                // LMS informing us that favorites have changed; request an update to the favorites list
+                requestFavorites();
+                return;
+            }
+            if (messageParts.length < 7) {
+                logger.trace("No favorites in message.");
+                return;
+            }
+
+            List<Favorite> favorites = new ArrayList<>();
+            String title = "";
```

Comment:
Where is this used for?

Reply:
I chose to capture the favorites title and count, as they are returned in the response from the LMS, even though they currently are not used anywhere. Would you rather they be removed, or commented as currently unused?

Reply:
Or you can log them inside a `isTraceEnabled` and not store it at all?